app/assets/stylesheets/layout/page_specific/_content_hub.scss

```diff
@@ -0,0 +1,234 @@
+// Content Hub Variables --
+// Colours
+$content-hub-title-color: $color-black;
+$content-hub-bgcolor: $color-grey-eight;
```

Comment:
Minor point but can we use `$color-grey-pale` - this will be the same colour as the breadcrumb header. We have waaaay too many different tones to pick from!